tests/e2e/network/service_annotations.go

```diff
@@ -238,13 +238,45 @@ var _ = Describe("Service with annotation", Label(utils.TestSuiteLabelServiceAnn
 		Expect(ret).To(BeTrue(), "external ip %s is not in the target subnet %s", ip, newSubnetCIDR)
 	})
 
+	It("should support service annotation 'service.beta.kubernetes.io/azure-load-balancer-enable-high-availability-ports'", func() {
+		if !strings.EqualFold(os.Getenv(utils.LoadBalancerSkuEnv), string(network.PublicIPAddressSkuNameStandard)) {
+			Skip("azure-load-balancer-enable-high-availability-ports only work with Standard Load Balancer")
+		}
+
+		annotation := map[string]string{
+			consts.ServiceAnnotationLoadBalancerEnableHighAvailabilityPorts: "true",
+			consts.ServiceAnnotationLoadBalancerInternal:                    "true",
+		}
+		// create service with given annotation and wait it to expose
+		ip := createAndExposeDefaultServiceWithAnnotation(cs, serviceName, ns.Name, labels, annotation, ports)
+		defer func() {
```

Comment:
> Please clean up pip as well

Actually it is an internal ip, maybe we don't need to clean it? https://github.com/kubernetes-sigs/cloud-provider-azure/blob/02a563ad60e356bcd66d8a5f702edae00643b087/tests/e2e/network/service_annotations.go#L227-L232

Reply:
Well. I don't think we need to clean it up but it is not because it is an internal ip. If we call something like `WaitCreatePIP`, we need to clean it up explicitly.

https://github.com/kubernetes-sigs/cloud-provider-azure/blob/941493e3a21707d6ecb54731c02a22989967f328/tests/e2e/network/service_annotations.go#L384

Reply:
> Well. I don't think we need to clean it up but it is not because it is an internal ip. If we call something like `WaitCreatePIP`, we need to clean it up explicitly.
> 
> https://github.com/kubernetes-sigs/cloud-provider-azure/blob/941493e3a21707d6ecb54731c02a22989967f328/tests/e2e/network/service_annotations.go#L384-L389

It makes sense to me.